### DISCUSSION TITLE
Switch desktops / displays?

### DISCUSSION THREAD

**Discussion by bigpalu** [Ross-Patterson/Portals-Desktop-Organization#108] [General]
Not sure if I am missing the option or not, but I don't seem to find a way to bring my portals to my main screen. While changing display settings in Windows 10 I mirrored my two displays, but after returning to the original display layout, my portals are stuck on the secondary display. 

My guess, one reason might be that my secondary gets the number 1 in the display settings and my main display the number two (due to the I/O differences´). But then again, isn't there any option to tell Portals where I want my layout XYZ on which display?

If I save the layout as a preset, delete the current layout and then load from preset, the display position remains. 

Thanks in advance :)

**Comment by bigpalu** [comment]
Or should the question be turned into a feature request?

**Comment by Ross-Patterson** [comment]
You can try installing the early testing version from here:
https://github.com/Ross-Patterson/Portals-Desktop-Organization/discussions/107

In v3.0.0.3, i have abandoned a lot of the layout mechanics in an attempt to keep it more simple and lightweight. Now you save a layout for each Display Setup you have, and Portals will automatically switch between them when it detects a change.

Try it and let me know how it works.
